bonsai/BudgetDetails/BudgetDetails.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  BudgetDetails.swift
+//  bonsai
+//
+//  Created by hoang on 07.12.2021.
+//
+
+import SwiftUI
+
+struct BudgetDetails: View {
+    var body: some View {
+        /*
+         Need to update when navigation bar will be added
+         */
+        VStack {
+            Spacer()
+            VStack(alignment: .leading) {
+                Spacer()
+                BudgetNameView()
+                    .frame(height: 33)
+                    .padding(.leading, 8)
+                // big square 48, image 34, padding 16, so leading is 8
+                
+                Text("Handsome, you’re doing well!")
+                    .font(BonsaiFont.body_15)
+                    .foregroundColor(BonsaiColor.text)
+                    .padding(.leading, 16)
+            }
+            HStack(alignment: .center, spacing: 16) {
+                BudgetMoneyTitleView()
+                    .padding(.leading, 16)
+                BudgetMoneyTitleView()
+            }
+            .frame(height: 63)
+            
+            ZStack {
+                Image(Asset.tree.name)
```

Comment:
image have padding from leading but it shouldn't

Reply:
fixed